### PR TITLE
fix(build): Use module-resolver for style imports

### PR DESCRIPTION
**Pull request by eckdanny-osi**
Using a module mapper (for local dev) allows us to stop _naively_ copying `[s]css` files from src to dest solely to preserve the relative import path. This duplication is still present in npm pkgs (though we can resolve by ignoring src files). The live-reloading in CRA is a bit quicker and there is less mgmt of update/add/remove scss file propagation to dist now.

closes [CL-64](https://osi-cwds.atlassian.net/browse/CL-64). Opened as an alternative to #474

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -4,4 +4,6 @@ libs/core/scss
 libs/core/js/core.js
 *.json5
 libs/*/dist
+libs/*/compiled
```
Comment (eckdanny-osi) [inline .eslintignore]: this should be in a separate PR

File: .eslintignore [hunk 2]
```diff
@@ -4,4 +4,6 @@ libs/core/scss
...
 **/node_modules/*
+/coverage
```
Comment (eckdanny-osi) [inline .eslintignore]: this should be in a separate PR

Comment (mlipardo) [reply .eslintignore]: 👍

File: libs/core/scss/mixins/_bg-checkers.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+@mixin bg-checkers {
```
Comment (eckdanny-osi) [inline libs/core/scss/mixins/_bg-checkers.scss]: remove

File: libs/core/scss/_mixins.scss [hunk 4]
```diff
@@ -1 +1,2 @@
 @import "mixins/scroll";
+@import "mixins/bg-checkers"
```
Comment (eckdanny-osi) [inline libs/core/scss/_mixins.scss]: remove

File: libs/docs/src/DemoCard/DemoCard.module.scss [hunk 5]
```diff
@@ -1,25 +1,6 @@
-// @import "@cwds/core/scss/custom";
+@import "~@cwds/core/scss/base";
```
Comment (eckdanny-osi) [inline libs/docs/src/DemoCard/DemoCard.module.scss]: remove

File: libs/docs/src/PropTable/PropTable.module.scss [hunk 6]
```diff
@@ -1,10 +1,12 @@
-// @import '~@cwds/core/scss/_base';
+@import "~@cwds/core/scss/base";
```
Comment (eckdanny-osi) [inline libs/docs/src/PropTable/PropTable.module.scss]: remove

File: libs/docs/src/PropTable/PropTable.module.scss [hunk 7]
```diff
@@ -1,10 +1,12 @@
...
 
 .PropDocs {
-  > .PropDoc {
-    border-top: 1px solid #f5f5f5;
-    &:first-child {
-      border-top: none;
-    }
+  border: 5px solid red;
```
Comment (eckdanny-osi) [inline libs/docs/src/PropTable/PropTable.module.scss]: remove

File: .eslintignore (hunk 2 above)
Comment (eckdanny-osi) [reply .eslintignore]: see #479

**Comment by eckdanny-osi** [comment]
spun off #479

File: libs/core/scss/mixins/_bg-checkers.scss (hunk 3 above)
Comment (eckdanny-osi) [reply libs/core/scss/mixins/_bg-checkers.scss]: moved to #480

File: libs/core/scss/_mixins.scss (hunk 4 above)
Comment (eckdanny-osi) [reply libs/core/scss/_mixins.scss]: moved to #480

File: libs/docs/src/DemoCard/DemoCard.module.scss (hunk 5 above)
Comment (eckdanny-osi) [reply libs/docs/src/DemoCard/DemoCard.module.scss]: moved to #480

File: libs/docs/src/PropTable/PropTable.module.scss (hunk 6 above)
Comment (eckdanny-osi) [reply libs/docs/src/PropTable/PropTable.module.scss]: moved to #480

**Comment by eckdanny-osi** [comment]
spun off #480

File: apps/www/package.json [hunk 8]
```diff
@@ -15,7 +15,7 @@
     "react": "^16.4",
     "react-dom": "^16.4",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.5",
+    "react-scripts": "2.1.8",
```
Comment (eckdanny-osi) [inline apps/www/package.json]: separate PR plz

File: libs/components/package.json [hunk 9]
```diff
@@ -25,10 +25,10 @@
     "build:esm": "babel --verbose --root-mode upward --out-dir dist src",
     "build:scss": "cpx --verbose \"src/**/*.{css,scss}\" dist",
     "build:cjs": "rollup -c",
+    "//postbuild:cjs": "sed -i bak \"s,'./,'../src/,g\" dist/index.cjs.js",
```
Comment (eckdanny-osi) [inline libs/components/package.json]: remove this

File: apps/www/package.json (hunk 8 above)
Comment (eckdanny-osi) [reply apps/www/package.json]: see #481

**Comment by eckdanny-osi** [comment]
spun off #481

**Comment by eckdanny-osi** [comment]
retest this please

**Comment by eckdanny-osi** [comment]
oh no's... jenkins, y u no status check?
